### DISCUSSION TITLE
2023/06/22 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_mecha#207] [Meeting Agendas]
- Multi-animal tracking
    - https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/260
    - https://github.com/SainsburyWellcomeCentre/aeon_mecha/pull/206
- 'aeon_experiments' codebase structure for future experiments
    - https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/196
    - https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/197
    - https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/186
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/262
- SciViz Video Streamer updates
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/175

**Comment by ttngu207** [comment]
Ingest data for the current, ongoing presocial experiments. We will discard these data later because these are primarily for testing the system, but let's make sure the ingestion is running as expected as well.
@JaerongA

**Reply by JaerongA** [reply]
Following up on this. 226 epochs and 2727 chunks have been ingested so far from `presocial0.1` from AEON2 and AEON3 (I don't see any data in AEON4 yet). 

Are you running any analysis on these datasets @jkbhagatio? I'm trying to see if there's any way I can validate the ingested data.

**Comment by ttngu207** [comment]
Dario is planning for a new 24/7 experiment 
- 2 animals
- Similar to the foraging experiment
- First week of july 2023
- May not need to ingest

**Comment by glopesdev** [comment]
Proposal for changing metadata into a more distributed format ("per-stream"):
  * Have one metadata file per stream describing the properties of the device/stream
  * In the future we can create links between external metadata files (e.g. Harp devices have their own schemas available publicly which could tell us the metadata of Harp device streams, column names, etc)
